### PR TITLE
No point in it being made inside of the function

### PR DESCRIPTION
**Pull request by SleepyMode**

### CODE REVIEW

**Comment by impulsh** [comment]
You’ll want to just call ScrW directly instead of using a variable because it will only be set once when the file is loaded and won’t change if the user changes their resolution

**Comment by alexgrist** [comment]
If possible try and keep your commit names to be descriptive of the change in future.

File: gamemode/core/libs/sh_notice.lua [hunk 1]
```diff
@@ -26,7 +26,14 @@ if (SERVER) then
 else
 	-- List of notice panels.
 	ix.notices = ix.notices or {}
-
+	
+	-- Move all notices to their proper positions.
+	local function OrganizeNotices()
+		for k, v in ipairs(ix.notices) do
```
Comment (MrMe0w) [inline gamemode/core/libs/sh_notice.lua]: it would benefit the performance to localize the scrW here, inside the function but outside of the loop

**Comment by SleepyMode** [comment]
Will do, cheers.

**Pull request merged**